src/EnrollWindowsHelloController.js

```diff
@@ -110,25 +125,73 @@ function (Okta, FormController, FormType, webauthn, Footer, WindowsHelloErrorMes
 
       formChildren: function () {
         var result = [];
+
         if (!webauthn.isAvailable()) {
           result.push(
             FormType.View(
-              { View: WindowsHelloErrorMessageView },
-              { selector: '.o-form-error-container' }
+              {View: WindowsHelloErrorMessageView},
+              {selector: '.o-form-error-container'}
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c">\
+              {{i18n code="enroll.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+        }
+
+        result.push(FormType.View({View: '<div class="okta-waiting-spinner hide"></div>'}));
```

Comment:
We have this same spinner for U2F, so this is to make Windows Hello use the same design. 
If I remember correctly the one you are talking @rchild-okta is for loading around the beacon, which is more from transition from one state to the other. The spinner here is to show that we are waiting for some external authorization to happen (either U2F or Windows Hello). 
I don't know the answer about Nico and Vince talking about it, but I would assume they did.